### PR TITLE
Unit Tests - Line chart

### PR DESCRIPTION
**Pull request by srmukher**
Added unit tests for Line chart increasing code coverage **from 70.42% to 90.41%**

Before:
image

After:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3f805b9bca38c866d56233f1b89868465b9d3bb0'>3f805b9bca38c866d56233f1b89868465b9d3bb0</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-ad375115-84a3-4b6d-b1f7-7549446649d3.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-ad375115-84a3-4b6d-b1f7-7549446649d3.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29049&commitId=8c8fddc49b40a2589affd9f8ea46287df03b4777&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.default.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by srmukher** [comment]
Closing this PR as the unit tests have been checked in via another PR.

**Pull request closed without merging**